### PR TITLE
DBVis: Add suggest java

### PR DESCRIPTION
**Pull request by linsui**
Close https://github.com/lukesampson/scoop-extras/issues/2244

### CODE REVIEW

File: bucket/dbvis.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
-    "homepage": "https://www.dbvis.com/",
     "version": "10.0.21",
+    "homepage": "https://www.dbvis.com/",
+    "description": "A universal database tool for developers, DBAs and analysts.",
+    "license": "Proprietary",
```
Comment (Ash258) [inline bucket/dbvis.json]: No url for proprietary license

Comment (linsui) [reply bucket/dbvis.json]: Could you tell me what do `Proprietary` and `Freeware` mean? I can't find an explanation in doc.

File: bucket/dbvis.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {
-    "homepage": "https://www.dbvis.com/",
     "version": "10.0.21",
+    "homepage": "https://www.dbvis.com/",
+    "description": "A universal database tool for developers, DBAs and analysts.",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.dbvis.com/purchase/"
```
Comment (Ash258) [inline bucket/dbvis.json]:
```suggestion
        "url": "https://www.dbvis.com/eula/"
```

File: bucket/dbvis.json (hunk 1 above)
Comment (Ash258) [reply bucket/dbvis.json]: https://en.wikipedia.org/wiki/Freeware

> There is no agreed-upon set of rights, license, or EULA that defines freeware unambiguously

https://en.wikipedia.org/wiki/Software_license#Proprietary_software_licenses

> The hallmark of proprietary software licenses is that the software publisher grants the use of one or more copies of software under the end-user license agreement (EULA), but ownership of those copies remains with the software publisher (hence use of the term "proprietary"). This feature of proprietary software licenses means that certain rights regarding the software are reserved by the software publisher. Therefore, it is typical of EULAs to include terms which define the uses of the software, such as the number of installations allowed or the terms of distribution.

Comment (linsui) [reply bucket/dbvis.json]: I got. Freeware does not means free in freedom. Thanks!

**Pull request merged**